frontend/src/metabase/setup/components/DatabaseStep/DatabaseStep.tsx

```diff
@@ -140,7 +140,6 @@ const DatabaseForm = ({
           <FormField name="engine" onChange={handleEngineChange} />
           <DriverWarning
             engine={values.engine}
-            hasBorder={true}
```

Comment:
Have you confirmed with designers that it's OK? I remember that there was a requirement to display the border in setup but not in admin.

Reply:
It was intentional on my part ... but I'm checking with product + design on when they want borders for all cases.

Reply:
talked with design, and decided to simply always use the border because it looks better and is more consistent with other design elements in the app.